### PR TITLE
adds favicon

### PR DESCRIPTION
**Pull request by quintus-luis**

### CODE REVIEW

**Review by Aloso — CHANGES_REQUESTED** [review]
I really like this. However, while this icon looks good on a dark background, it's difficult to see on white. Would you mind adding a dark background to the icon? I'm thinking of this color: `#100522`

**Comment by quintus-luis** [comment]
Good Point!

File: index.html [hunk 1]
```diff
@@ -17,6 +17,7 @@
     <meta property="og:url" content="https://wewanttosleep.de/"/>
 
     <link rel="apple-touch-icon" href="/logo192.png" />
+    <link rel="mask-icon" href="favicon.svg">
```
Comment (94726) [inline index.html:20]:
```suggestion
    <link rel="mask-icon" href="/favicon.svg">
```

**Review by 94726 — CHANGES_REQUESTED** [review]

**Review by 94726 — APPROVED** [review]

**Review by Aloso — CHANGES_REQUESTED** [review]
`logo256.png` is 512x512 pixels, please rename it to `logo512.png`.

Since you renamed `manifest.json` to `site.webmanifest`, you need to change it in the `index.html` as well.

I personally don't like that the manifest is no longer formatted. It's not worthwhile to make a file unreadable just to save a few bytes (if that was your intention).

**Comment by LeoTuet** [comment]
> `logo256.png` is 512x512 pixels, please rename it to `logo512.png`.
> 
> Since you renamed `manifest.json` to `site.webmanifest`, you need to change it in the `index.html` as well.
> 
> I personally don't like that the manifest is no longer formatted. It's not worthwhile to make a file unreadable just to save a few bytes (if that was your intention).

I overlooked that - all

**Review by Aloso — APPROVED** [review]
LGTM

**Pull request merged**